packages/components/src/Link/Link.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { Link } from ".";
+
+afterEach(cleanup);
+
+it("renders a Link", () => {
+  const { container } = render(<Link text="Foo" url="https://getjobber.com" />);
+  expect(container).toMatchSnapshot();
```

Comment:
Maybe these came from the generator... (if so, we should probably update to not use snapshots)...
There's a better way to test these using React Testing Library... I'll try to find some examples in our code and link them here.

Reply:
Appreciate that - and yeah, I just kept the basics from the template, and removed one that felt unapplicable.

Reply:
Page has some tests for when links appear in the intro, they are probably a good pattern to follow: 
https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Page/Page.test.tsx#L144-L191